unrouted_handler.go

```diff
@@ -115,7 +119,7 @@ func NewUnroutedHandler(config Config) (*UnroutedHandler, error) {
 	}
 
 	// Only promote extesions using the Tus-Extension header which are implemented
-	extensions := "creation,creation-with-upload"
+	extensions := "creation,creation-with-upload,creation-defer-length"
```

Comment:
`creation-defer-length` should only be added if the data storage support it, similar to how the Termination extension is handled in the next lines.